### DISCUSSION TITLE
Development on temporary pause

### DISCUSSION THREAD

**Discussion by kamakazikamikaze** [easysnmp/easysnmp#170] [Announcements]
Unfortunately I woke up Saturday morning to discover that my 2014 MBP has experienced a terminal case of swollen battery. Though I've successfully disabled encryption on the drive, it will be a bit before a 12+16 pin SSD enclosure gets delivered. My development VM, working notes, and debugging scripts are all on the drive.

I have made a tough decision to not replace the device with another Apple product. Instead, I intend to use a Linux distro and hope to virtualize both macOS and Windows. I do not intend to abandon development for macOS but do understand that this may complicate things moving forward.

**Comment by amotl** [comment]
Dear Kent,

let me know if I can lend a hand to support you when it comes to testing builds on macOS. I am on macOS Catalina 10.15.7. Also, good luck recovering your machine and switching (back?) to Linux. My development workstation will probably follow the same path at some time in the future.

With kind regards,
Andreas.

**Comment by kamakazikamikaze** [comment]
I've created a VM on my old workstation, upgraded to Monterey 12.5, installed most tools, and then successfully caused a full crash when attempting to execute `python setup.py build -g`. But hey, progress, right?

**Reply by kamakazikamikaze** [reply]
Development appears to be able to resume this week
